enterprise/server/workflow/config/config.go

```diff
@@ -18,6 +18,10 @@ const (
 	// FilePath is the path where we can expect to locate the BuildBuddyConfig
 	// YAML contents, relative to the repository root.
 	FilePath = "buildbuddy.yaml"
+
+	// KytheActionName is the name used for actions automatically
+	// run by us if code search is enabled.
+	KytheActionName = "AutoKythe"
```

Comment:
this will show up in invocation history - idk if you wanted to call it something more obvious to users like "Build source code indexes" or something

(yeah, kind of ugly that the UI label doubles as the action's "ID", but ... :shrug: )

Reply:
Done.